### DISCUSSION TITLE
update labels / translations for neg-w-call and neg-wo-call

### DISCUSSION THREAD

**Discussion by mdewinne** [OP-TED/eForms-SDK#855] [Q&A] [answered]
In https://github.com/OP-TED/eForms-SDK/blob/develop/translations/code_en.xml the labels for `neg-w-call` and `neg-wo-call` could be updated to reflect the correct procedure name according to the applicable legal basis (which can be derived from the notice subtype).

Example: in https://github.com/OP-TED/eForms-SDK/blob/develop/translations/code_en.xml#L16790
    - `neg-w-call`, subtype 16 = Dir 2014/24/EU
    - current label reads _Negotiated with prior publication of a call for competition / competitive with negotiation_ and thus also contains the procedure name for the utilities directive, which is confusing for some users when they see this in the notice editor or the HTML- or PDF-export of the notice
    - correct label for that legal basis would be _Competitive procedure with negotiation_
 
So, depending on the legal basis of the notice subtype, the label of

- `neg-w-call`
     - should read _Competitive procedure with negotiation_ (for Dir 2014/24/EU)
     - should read _Negotiated procedure with prior call for competition_ (for Dir 2014/25/EU)
     - should read _Negotiated procedure with publication of a contract notice_ (for Dir 2009/81/EC)
- `neg-wo-call`
     - should read _Negotiated procedure without prior publication_ (for Dir 2014/24/EU)
     - should read _Negotiated procedure without prior call for competition_ (for Dir 2014/25/EU)
     - should read _Negotiated procedure without publication of a contract notice_ (for Dir 2009/81/EC)

**Comment by YvesJo** [comment, marked as answer]
Hi,
This would certainly require a review of the codelist and would get rid of the commonalities considered at codelist creation. The concept of Contract Notice has also been replaced by the one of Competition Notice with eForms.
We need to discuss this internally and will get back to you.

**Reply by YvesJo** [reply]
Hi again,
We have tailored codelists defined, like it is the case for Procurement Procedure Type with one tailored list per notice subtype. These lists however share, for a given code, the same label as the one of the parent list.
Reviewing the Procurement Procedure Type codelist is something that goes beyond the SDK and that you should discuss with DG GROW.

**Reply by mdewinne** [reply]
OK thank you for your answer!
